projects/storefrontlib/src/layout/direction/config/direction.config.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { Direction } from './direction.model';
+
+/**
+ * The direction config provides an easy way to configure "ltr" versus "rtl" direction
+ * for the storefront. The direction can be configured to detect the direction by language.
+ *
+ * The following configuration detects rtl languages by isoCode for Arabic and Hebrew:
+ *
+ * ```typescript
+ * direction: {s
+ *   detect: true,
+ *   default: DirectionMode.LTR,
+ *   rtlLanguages: ['ar', 'he']
+ * }
+ * ```
+ */
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class DirectionConfig {
```

Comment:
Bagatelle: Since it became a separate config token, for customer's convenience we can consider adding it to the `StorefrontConfig` type.

https://github.com/SAP/spartacus/blob/103e3ae/projects/storefrontlib/src/storefront-config.ts#L29